dmoj/tests/test_checker.py

```diff
@@ -4,79 +4,87 @@
 
 
 class CheckerTest(unittest.TestCase):
-    def assert_pass(self, check, str1, str2, expect=True):
+    def assert_standard_pass(self, check, str1, str2, expect=True):
         if expect:
             message = 'expecting %r to equal %r'
         else:
             message = 'expecting %r to not equal %r'
         self.assertEqual(bool(check(str1, str2)), expect, message % (str1, str2))
         self.assertEqual(bool(check(str2, str1)), expect, message % (str2, str1))
 
-    def assert_fail(self, check, str1, str2):
-        self.assert_pass(check, str1, str2, expect=False)
+    def assert_standard_fail(self, check, str1, str2):
+        self.assert_standard_pass(check, str1, str2, expect=False)
 
     def test_standard(self):
         from dmoj.checkers.standard import check
 
-        self.assert_pass(check, b'a', b'a')
-        self.assert_pass(check, b'a b', b'a  b')
-        self.assert_pass(check, b'a b   \n', b'a b')
-        self.assert_pass(check, b'\n\na b \n    ', b'a b')
-        self.assert_pass(check, b'a\n\n\nb', b'a\nb')
-        self.assert_pass(check, b'  a   \n\n', b'\n\n\n  a   \n')
-        self.assert_pass(check, b'a ' * 1000, b' a' * 1000)
-        self.assert_pass(check, b'a\n' * 1000, b'\n\n\na' * 1000)
+        self.assert_standard_pass(check, b'a', b'a')
+        self.assert_standard_pass(check, b'a b', b'a  b')
+        self.assert_standard_pass(check, b'a b   \n', b'a b')
+        self.assert_standard_pass(check, b'\n\na b \n    ', b'a b')
+        self.assert_standard_pass(check, b'a\n\n\nb', b'a\nb')
+        self.assert_standard_pass(check, b'  a   \n\n', b'\n\n\n  a   \n')
+        self.assert_standard_pass(check, b'a ' * 1000, b' a' * 1000)
+        self.assert_standard_pass(check, b'a\n' * 1000, b'\n\n\na' * 1000)
 
-        self.assert_fail(check, b'a', b'b')
-        self.assert_fail(check, b'\n\n\na \n b\n', b'a b')
-        self.assert_fail(check, b'a\n\n\nb', b'a b')
-        self.assert_fail(check, b'ab', b'a b')
+        self.assert_standard_fail(check, b'a', b'b')
+        self.assert_standard_fail(check, b'\n\n\na \n b\n', b'a b')
+        self.assert_standard_fail(check, b'a\n\n\nb', b'a b')
+        self.assert_standard_fail(check, b'ab', b'a b')
 
         # Checkers should handle mixed bytes/str
-        self.assert_pass(check, b'a', u'a')
-        self.assert_fail(check, b'a', u'b')
+        self.assert_standard_pass(check, b'a', u'a')
+        self.assert_standard_fail(check, b'a', u'b')
+
+    def assert_presentation_error(self, result, feedback):
```

Comment:
Shouldn't this be named something like `assert_wa_and_feedback`? (since WA + has feedback != presentation error)